src/cmd/top/joblist_pane.c

```diff
@@ -62,6 +65,10 @@ static json_t *get_current_job (struct joblist_pane *joblist)
 }
 
 
+static bool show_queue (struct joblist_pane *joblist)
+{
+}
```

Comment:
you have this in the ` flux-mini: remove -q short option for --quiet ` commit, I guess a merge conflict.